build/Jenkinsfile

```diff
@@ -3,28 +3,53 @@ pipeline {
     label 'jenkins-slave'
   }
   options {
-    buildDiscarder(logRotator(numToKeepStr: '13'))
-    disableConcurrentBuilds()
+    buildDiscarder(logRotator(daysToKeepStr: '6'))
+  }
+  parameters {
+    string(
+        name: 'rds_instance_name',
+        description: 'Enter a name for the DoS DB RDS instance',
+    )
+    choice(
+        name: 'build_action',
+        choices: 'create\ndestroy',
+        description: 'Enter whether your looking to create an DoS DB RDS instance or delete one'
+    )
   }
   stages {
     stage("Show Variables") {
       steps {
-        script {
-          sh 'make devops-print-variables'
-        }
+        sh 'make devops-print-variables'
+      }
+    }
+
+    stage('Create RDS Instance'){
+      when {
+        expression { params.build_action == 'create' }
+      }
+      steps {
+        sh 'make docker-config'
+        sh 'make create-instance INSTANCE_NAME=${rds_instance_name}'
       }
     }
-    stage("Run Test Suite") {
+
+    stage('Populate Database'){
+      when {
+        expression { params.build_action == 'create' }
+      }
+      steps {
+        sh 'export DB_PASSWORD=$(make secret-fetch NAME=${rds_instance_name}-nonprod-dos_db_password) && make populate-database INSTANCE_NAME=${rds_instance_name}'
```

Comment:
This is unnecessary complicated. Please, can we wrap it in a single make target?